plugins/sql/komand_sql/actions/query/action.py

```diff
@@ -12,7 +13,7 @@ def __init__(self):
         )
 
     def run(self, params={}):
-        with self.connection.SQLConnection(self.connection.conn_str) as s:
+        with connection.SQLConnection(self.connection.conn_str) as s:
```

Comment:
This seems wrong. Why did you remove the 'self' reference there?

Reply:
@jmcadams-r7 I removed this because the action didn't work because it couldn't find the SQLConnection class.

Reply:
Right, right...hmmm...It's ok I think. The With line is going to create a new instance of that class on the fly and destroy it as it comes out of the with.